pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto

```diff
@@ -166,12 +166,12 @@ package v1alpha2;
 
 // IstioControlPlane is a CustomResourceDefinition (CRD) describing an Istio control plane.
 message IstioControlPlane {
+    // Defines the object meta
+    k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
```

Comment:
this is correct, but there should be another field added to `IstioControlPlane`.  Unfortunately I am not quite sure what it is - TypeMeta iirc...

Reply:
This approach is not going to work, unless we abandon proto as a source of truth (which we discussed with @geeknoid in the past). There's no way to create an anonymous field in go structs from proto. 
Please see https://github.com/istio/operator/pull/370 for workaround.

Reply:
I’ll include #370 in this PR and get it in working with order post KUBECON recovery.

Reply:
I’ll include #370 in this PR and get it in working with order post KUBECON recovery.